Framework/API/test/AsynchronousTest.h

```diff
@@ -143,12 +156,18 @@ class AsynchronousTest : public CxxTest::TestSuite {
   void testCancelGroupWS() {
     WorkspaceGroup_sptr groupWS = makeGroupWorkspace();
     AsyncAlgorithm alg;
+    alg.synchronise = true;
```

Comment:
Is this correct? `synchronise` is set to true on the parent object but not on the child algorithms so I don't think the `if` statement above will ever fire?